### DISCUSSION TITLE
Email functions

### DISCUSSION THREAD

**Discussion by tyaoning** [Leantime/leantime#519] [General]
Hello 

I saw the documentation said, we can configure the email settings in configuration.php file. However, there is no any information about how email functions works. For example, when email will be sent to whom, what are the triggers?

If the functions are not ready yet, I hope can be added in next version, that will make this amazing project even better.

Thanks

**Comment by marcelfolaron** [comment]
Thanks for the feedback. That is something we need to add to our documentation. Here a quick summary of the top of my head:

In general only users assigned to a project will get those project updates:

- Any new or updated To-Dos
- New comments anywhere
- New/updated ideas, retrospective items, lean canvas items
- New/updated milestones

**Comment by csimstewart** [comment]
We only seem to have the forgot password email function working. Is there somewhere else we configure the email to send updates via email to team members mentioned above?

**Comment by Phantom4us** [comment]
Is it possible to set up LeanTime to check an email address to download emails and add tasks that are emailed into the system? 
What I've used a lot on other to-do list programs is if I get an email I need to take care of but not right now, I forward it to a special email address that then ends up in an "Inbox" on my To-Do list.   Is that functionality possible?

**Comment by marcelfolaron** [comment]
Hi,

That is currently not a feature in Leantime but I‘d love to see it.

I can do some quick research and see what we would need to do. I don’t
think it’s actually that much.

**Comment by HardNETWD** [comment]
Hello.
When a project is created, the system sends emails to all participants. (Good).
But when I create a task, I would like only the user assigned to the task to receive mail, and not all project members. This generates many emails that hinder the rest of the members.
Is it possible to configure this?
thank you

**Comment by marcelfolaron** [comment]
Hi,

Thanks for the feedback. That is a good point. The initial thought behind
notifying everyone is to ensure someone takes ownership. But if it is
assigned then this might not be an issue.

I guess we could send an email to everyone if it has not been assigned to
anyone and send emails only to the owner if it is assigned.

**Reply by PerseusAT** [reply]
Hi.
First: I love leantime! It's almost perfect. Not too simple, not too big and complicated! Congrats, good job! Please keep it simple.
I think that with the emails is an important topic. Everyone should be able to set this themselves. Everyone thinks a little differently ;-).

**Comment by hemantbabu** [comment]
Hi,
I am not a programmer and my questions could be stupid. Apologies...
Which file should I insert the SMTP details for the mail functions? In my installation I can see:
/var/www/leantime/config/conifiguraiton.sample.php
Thank you in advance for your guidence and of course big thanks for such a well thought out project management system for people like me

**Reply by gloriafolaron** [reply]
Hi! Did you end up getting this figured out? As a heads up, in the future, support responses can be quicker through our Discord: https://discord.gg/sugXFb79Hn
